src/component-library/HelperText/HelperText.tsx

```diff
@@ -0,0 +1,47 @@
+import { forwardRef, HTMLAttributes } from 'react';
+
+import { StyledHelperText, StyledSubHelperText } from './HelperText.style';
+
+// checks for thruthy messages both for string and string[]
+const hasErrorMessage = (errorMessage?: string | string[]): boolean =>
+  typeof errorMessage === 'string' ? !!errorMessage : !!errorMessage?.filter(Boolean).length;
+
+type Props = {
+  errorMessage?: string | string[];
+  errorMessageProps?: HTMLAttributes<HTMLElement>;
```

Comment:
Did we discuss getting rid of the errorMessageProps and descriptionProps?

Reply:
I removed it from being an exposed prop of the `Input`.
I have it here for the integration with `react-aria`, because it's exactly what they provide us with.

Reply:
`const { inputProps, descriptionProps, errorMessageProps, labelProps } = useTextField(props, inputRef);`